nixos/modules/programs/java.nix

```diff
@@ -5,11 +5,9 @@
 
 with lib;
 
-let
-  cfg = config.programs.java;
-in
+let cfg = config.programs.java;
 
-{
+in {
```

Comment:
Was there a reason this was re-formatted?

Reply:
Not sure, I think I ran a nixfmt command but I don't remember.

Reply:
I see the original styling that was there more, but I also don't really care much either way